apps/omg_watcher/test/omg_watcher/exit_processor/standard_exit_test.exs

```diff
@@ -56,14 +56,32 @@ defmodule OMG.Watcher.ExitProcessor.StandardExitTest do
   end
 
   describe "Core.determine_standard_challenge_queries" do
+    test "doesn't ask for anything and stops if deposit utxo not spent at all",
+         %{alice: alice, processor_empty: processor} do
+      processor = processor |> start_se_from_deposit(@utxo_pos_deposit, alice)
```

Comment:
yes, I know, single pipes. But before this is raised, please observe how consistent and aesthetically uniform this allows this file to be. Style makes us to avoid **"needless pipe" (not "single")**, I consider this a non-needless pipe (from the perspective of the entire file's context).

If this is, regardless of the argument given, deemed unacceptable, then <sigh>... I can ofc adjust

Reply:
How dare you @pdobacz you betrayed __The Convention__ we all agreed on :exclamation: 
[crying profusely :cry: not really want the fix]

Reply:
I'm serious, I want to follow the style, I'm just making an argument here: I think this is warranted and within the style limits.

Reply:
I havn't made up my mind on this yet. Some places single pipes make sense, other places make it harder to read.

But in this case, I like how it can be read as a natural sentence so I'm letting this through.